kfs-purap/src/main/java/edu/arizona/kfs/module/purap/businessobject/CreditMemoItem.java

```diff
@@ -0,0 +1,78 @@
+package edu.arizona.kfs.module.purap.businessobject;
+
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.Iterator;
+
+import org.kuali.kfs.module.purap.businessobject.AccountsPayableItemBase;
+import org.kuali.kfs.module.purap.businessobject.CreditMemoAccount;
+import org.kuali.kfs.module.purap.businessobject.PaymentRequestAccount;
+import org.kuali.kfs.module.purap.businessobject.PaymentRequestItem;
+import org.kuali.kfs.module.purap.businessobject.PurchaseOrderItem;
+import org.kuali.kfs.module.purap.document.VendorCreditMemoDocument;
+import org.kuali.kfs.module.purap.document.service.AccountsPayableService;
+import org.kuali.kfs.module.purap.util.ExpiredOrClosedAccountEntry;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.kuali.rice.krad.util.ObjectUtils;
+
+public class CreditMemoItem extends org.kuali.kfs.module.purap.businessobject.CreditMemoItem {
+    
+    public CreditMemoItem() {
```

Comment:
In attempting to remove this default Constructor, an error continues to occur:

SEVERE: Exception sending context initialized event to listener instance of class org.kuali.kfs.sys.context.KFSInitializeListener
org.kuali.rice.krad.datadictionary.DataDictionaryException: Exception validating BusinessObjectEntry for class edu.arizona.kfs.module.purap.businessobject.CreditMemoItem
	at org.kuali.rice.krad.datadictionary.BusinessObjectEntry.completeValidation(BusinessObjectEntry.java:86)
	at org.kuali.rice.kns.datadictionary.BusinessObjectEntry.completeValidation(BusinessObjectEntry.java:30)
	at org.kuali.rice.krad.datadictionary.DataDictionary.validateDD(DataDictionary.java:285)
	at org.kuali.rice.krad.config.KRADConfigurer.loadDataDictionary(KRADConfigurer.java:116)
	at org.kuali.rice.krad.config.KRADConfigurer.onApplicationEvent(KRADConfigurer.java:76)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:97)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:324)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:929)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:467)
	at org.kuali.rice.core.web.listener.KualiInitializeListener.contextInitialized(KualiInitializeListener.java:91)
	at org.kuali.kfs.sys.context.KFSInitializeListener.contextInitialized(KFSInitializeListener.java:35)
	at org.apache.catalina.core.StandardContext.listenerStart(StandardContext.java:4939)
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5434)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150)
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1559)
	at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1549)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)

While I cannot seem to find anywhere that calls this method, it's inclusion alleviates the error.